### PR TITLE
Change the data type for metadata and content fields in v1alpha1.CacheSizeSettings

### LINKED ISSUE #2231: Change the data type from string to *int in cacheSizeSettings field of repositoryserver CRD
**Is your feature request related to a problem? Please describe.**
```
// CacheSizeSettings are the metadata/content cache size details
// that can be used while establishing connection to the kopia repository
type CacheSizeSettings struct {
	Metadata string `json:"metadata"`
	Content  string `json:"content"`
}

```
Currently the `Metadata` and `Content` fields in `CacheSizeSettings` have string as a data type whereas kopia CLI accepts the values for these fields as integer.  

**Describe the solution you'd like**
It would be better if the CRD accepts the fields in `Integer` format since that would reduce the number of conversions required in the GO code

### PR DESCRIPTION
**Pull request by kale-amruta**
## Change Overview

```
// CacheSizeSettings are the metadata/content cache size details
// that can be used while establishing connection to the kopia repository
type CacheSizeSettings struct {
	Metadata string `json:"metadata"`
	Content  string `json:"content"`
}

```

Currently the Metadata and Content fields in CacheSizeSettings have string as a data type whereas kopia CLI accepts the values for these fields as integer.

It would be better if the CRD accepts the fields in Integer format since that would reduce the number of conversions required in the GO code

This PR changes the field types to `*int` for `metadata` and `content` fields 

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues 

- fixes #2231 

## Test Plan

- [ ] :muscle: Manual
- [x] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: pkg/customresource/repositoryserver.yaml [hunk 1]
```diff
@@ -44,12 +44,9 @@ spec:
                       to the kopia repository
                     properties:
                       content:
-                        type: string
+                        type: integer
```
Comment (ankitjain235) [inline pkg/customresource/repositoryserver.yaml:47]: Should we add description, about what unit would be used for the provided value?

**Review by ankitjain235 — COMMENTED** [review]
Added minor comment, looks good otherwise.

File: pkg/customresource/repositoryserver.yaml (hunk 1 above)
Comment (kale-amruta) [reply pkg/customresource/repositoryserver.yaml:50]: makes sense, I have added it

**Review by ankitjain235 — COMMENTED** [review]
LGTM.

**Review by PrasadG193 — APPROVED** [review]

**Pull request merged**